types/yaireo__tagify/index.d.ts

```diff
@@ -1602,6 +1608,31 @@ declare class Tagify<T extends Tagify.BaseTagData = Tagify.TagData> {
      */
     setDisabled(disabled: boolean): void;
 
+    /**
+     * Get data for the specific instance by `key` parameter from `localStorage`.
+     * @param key `localStorage` key (under the tagify namespace).
+     * @returns Data stored under `key` in `localStorage`.
+     * Returns `undefined` if {@link TagifyConstructorSettings.id} has not been set or no entry exists for `key` in `localStorage`.
+     */
+    getPersistedData(key: string): any;
```

Comment:
Can you change the return type to `unknown`? `any` should be avoided if possible, it easily pollutes other variables and effectively turns off type checks.